example/README.md

```diff
@@ -8,6 +8,11 @@ To run this example:
 
 4. Run the python entrypoint: `python main.py`
 
+or 
+
+1. Start Postgres and PostgREST `docker-compose up`
+2. Run the python entrypoint: `python main.py`
```

Comment:
Should this run under `docker compose` too?

Reply:
Could do that if you like. The idea was just to make it easy for people to play with the example after checking it out, so having main.py inside a container just adds steps that imo.